Makefile

```diff
@@ -164,5 +167,13 @@ stylesheets: public/css/index.css
 public/css/index.css: $(STYLESHEETS)
 	lessc $< $@
 
+.PHONY: swagger-ui
+swagger-ui:
+	rm -Rf public/assets/swagger-ui
+	git clone --depth=10 https://github.com/swagger-api/swagger-ui.git /tmp/swagger-ui
+	mv /tmp/swagger-ui/dist public/assets/swagger-ui
+	rm -Rf /tmp/swagger-ui
```

Comment:
Just download a tar.gz release instead. We shouldn't depend on latest master being functional